lms/services/__init__.py

```diff
@@ -72,7 +72,7 @@ def includeme(config):
         "lms.services.assignment.factory", name="assignment"
     )
     config.register_service_factory(
-        "lms.services.vitalsource.factory", name="vitalsource"
+        "lms.services.vitalsource.service_factory", name="vitalsource"
```

Comment:
Could we register by `iface` too? I mentioned that on the other PR. We could do that on a follow up PR and discuss it there.